acceptance/tests/apply/puppet_apply_graph.rb

```diff
@@ -1,5 +1,9 @@
 test_name 'puppet apply should generate a graph'
 
+tag 'audit:low',
+    'audit:integration',  # Core testing of `vardir` should occur in another test.
```

Comment:
This test does not test the graph output only that a file is created. Such a validation can be covered at the integration level.